apiData_arff.js

```diff
@@ -0,0 +1,71 @@
+var fs = require('fs');
+var XMLHttpRequest = require("xmlhttprequest").XMLHttpRequest;
+var url = 'http://pokedata.c4e3f8c7.svc.dockerapp.io:65014/api/pokemon/sighting/';
+
+var destination = 'arff/apiData.arff';
+
+httpGetAsync(url, destination, 'pokemonID');
+
+function httpGetAsync(url, destination, classKey) {
+    var xmlHttp = new XMLHttpRequest();
+    xmlHttp.onreadystatechange = function () {
+        if (xmlHttp.readyState == 4 && xmlHttp.status == 200) {
+            try {
+                var apiData = JSON.parse(xmlHttp.responseText);
+            } catch (e) {
+                console.log(e.stack);
+            }
+            jsonToArff(apiData, classKey, 'apiData', destination);
+        }
+    };
+    xmlHttp.open("GET", url, true);
+    xmlHttp.send();
+    return xmlHttp.responseText;
+}
+
+function jsonToArff(json_data, classKey, fileName, destination) {
+    var classLabels = allValuesForKeyInData(classKey, json_data);
+    var arff = '';
+    arff = arff + '@RELATION ' + fileName + '\n' + '\n';
+
+    var firstElement = json_data[0];
+    for (var key in firstElement) {
+        if (key !== classKey) {
+            if (typeof firstElement[key] === 'string') {
+                arff = arff + '@ATTRIBUTE ' + key + ' string\n';
+            }
+            if (typeof firstElement[key] === 'number'
+                || typeof firstElement[key] === 'boolean') {
+                arff = arff + '@ATTRIBUTE ' + key + ' numeric\n';
+            }
```

Comment:
we need to handle objects here as well, e.g. latitude and longitude are sent within a location object
`"location": {
            "coordinates": [
                -121.88805705004,
                37.3339857442005
            ],
            "type": "Point"
        }`
this needs to be considered for the attribute header and bellow for the data. However, I think we do not have to handle this case generic, as we do not know how other objects will look like (if there will be more).